### PR TITLE
Need to get arrays printed nicely!

### PR DESCRIPTION
**Pull request by leknaat**
In this state, it's working -- But I'd like to make a component called UseBy in junctionAbility that takes an array, unlike how I have useBy set up here.

It's also ugly as sin, but working ;)

### CODE REVIEW

File: src/components/hello/hello.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+export default class Hello extends React.Component {
+
+  render() {
+    return (<div><p>Hello World!</p><p>I am a basic greeter to make sure that all the routing is a-ok!</p></div>)
+  }
+}
```
Comment (JohnAGill) [inline src/components/hello/hello.js]: Is this used any where? If not we don't need it anymore

File: src/components/hello/index.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import React from 'react'
+import Hello from './hello'
+
+export default () => <Hello />
```
Comment (JohnAGill) [inline src/components/hello/index.js]: same as above if it's not used then remove it

File: src/components/junctionAbilities/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import Ability from '../junctionAbility/ability.js'
+import _ from 'lodash'
+
+export default class JunctionAbilities extends React.Component {
+
+
+  render(){
+    return (
+      <div>
+        {_.map(this.props.abilities, (ability) => {
```
Comment (JohnAGill) [inline src/components/junctionAbilities/index.js]: can you move this map out into a function call to make it clearer

File: src/components/junctionAbilitiesPage/index.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import JunctionAbilities from '../junctionAbilities'
+
+const abilitiesArray = [
```
Comment (JohnAGill) [inline src/components/junctionAbilitiesPage/index.js]: Lets move this into it's own file so and import it from there

File: src/components/hello/hello.js (hunk 1 above)
Comment (leknaat) [reply src/components/hello/hello.js]: Nope -- It's gone now. :)

File: src/components/hello/index.js (hunk 2 above)
Comment (leknaat) [reply src/components/hello/index.js]: Same as above.

File: src/components/junctionAbilities/index.js (hunk 3 above)
Comment (leknaat) [reply src/components/junctionAbilities/index.js]: Done!

File: src/components/junctionAbilitiesPage/index.js (hunk 4 above)
Comment (leknaat) [reply src/components/junctionAbilitiesPage/index.js]: Also done.

**Pull request merged**